modules/home-environment.nix

```diff
@@ -551,7 +551,7 @@ in
           text = ''
             # Only source this once.
```

Comment:
in all these files, you could also change this comment to: `Only source this once per shell`